### DISCUSSION TITLE
Is the NLP part of the chatbot working?

### DISCUSSION THREAD

**Discussion by Datcomx2** [hubtype/botonic#2605] [General]
I am using this framework and I would like to use the NLP part of the chatbot but I get a dependency error for tensorflow. I am also working on the MacBook Pro with the Apple M1 in Monterey. If you need more details please let me know thanks!

**Comment by jorge-murillo** [comment]
Did you find a solution? I'm trying to get the project running and connect it to WhatsApp but I can't find any support

**Reply by Datcomx2** [reply]
No, I haven't yet. The documents are outdated which is an issue.
